src/components/ToastUpdate.test.tsx

```diff
@@ -150,4 +150,31 @@ describe('ToastUpdate tests', () => {
     expect(restartButton).toBeEnabled()
     expect(dismissButton).toBeEnabled()
   })
+
+  test('Happy path: external links render correctly', () => {
+    const releaseNotesWithBreakingChanges = `
```

Comment:
[nit] should probably be named `releaseNotesWithLink`

Reply:
Shoot I had automerge on. Well, next time we edit that test file